cmd/proxy/actions/app_proxy.go

```diff
@@ -65,7 +66,5 @@ func addProxyRoutes(
 	handlerOpts := &download.HandlerOpts{Protocol: dp, Logger: l}
 	download.RegisterHandlers(r, handlerOpts)
 
-	storage.RegisterHandlers(r, s)
```

Comment:
This is inline with download protocol, would not the same apply download protocol as well.

I would still namespace into storage actions as an empty struct{} type.

Reply:
```
type storage struct{}
func (s * storage) RegisterHandlers(r, s)
```

Which makes it clear that the download protocol is not even involved. but this is a minor change that I won't worry about it as much.